causal_testing/testing/estimators.py

```diff
@@ -168,31 +169,53 @@ def estimate_control_treatment(self) -> tuple[pd.Series, pd.Series]:
         x = x[model.params.index]
 
         y = model.predict(x)
-        return y.iloc[1], y.iloc[0]
+
+        # Delta method confidence intervals from
+        # https://stackoverflow.com/questions/47414842/confidence-interval-of-probability-prediction-from-logistic-regression-statsmode
+        cov = model.cov_params()
+        gradient = (y * (1 - y) * x.T).T  # matrix of gradients for each observation
+        std_errors = np.array([np.sqrt(np.dot(np.dot(g, cov), g)) for g in gradient.to_numpy()])
+        c = 1.96  # multiplier for confidence interval
```

Comment:
Just double checking that this is a constant that will never change and therefore okay to hardcode?